uPortal-webapp/src/main/webapp/WEB-INF/jsp/Search/searchLauncher.jsp

```diff
@@ -32,16 +32,17 @@
 <div id="webSearchContainer" class="fl-widget">
     <div class="fl-widget-inner">
       <div class="fl-widget-content">
-        <c:set var="searchLabel"><spring:message code="search"/></c:set>
         <form class="form-search" role="form" method="POST" action="${searchLaunchUrl}" id="${n}webSearchForm">
           <div class="input-group">
             <spring:message code="search" var="searchPlaceholder" />
             <spring:message code="search.terms" var="searchTitle" />
-            <spring:message code="submit" var="searchSubmit" />
-            <input id="${n}webSearchInput" class="searchInput input-large search-query form-control" value="" name="query" type="search" placeholder="${searchPlaceholder}" title="${searchTitle}"/>
+            <spring:message code="search.submit" var="searchSubmit" />
+            <input id="${n}webSearchInput" class="searchInput input-large search-query form-control" value="" name="query"
+             type="search" aria-label="${searchTitle}" placeholder="${searchPlaceholder}" title="${searchTitle}"/>
```

Comment:
In general screen readers respond better to having a `<label>` connected to an input.
Since this label would need to be hidden to the user, class `sr-only` could be used.

```suggestion
            <label for="${n}webSearchInput" class="sr-only">${searchTitle}</label>
            <input id="${n}webSearchInput" class="searchInput input-large search-query form-control" value="" name="query"
             type="search" placeholder="${searchPlaceholder}" title="${searchTitle}"/>
```